microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -73,7 +73,7 @@ Go to the [developer dashboard](https://partner.microsoft.com/dashboard/microsof
 
 Use the **Packages** webpage to upload the zip file of your extension package.  You can only upload one package at a time.  Your submission is blocked if the package upload isn't successful on the **Packages** webpage.
 
-To upload the package, select and drag the package onto the upload field.  Also, you can select **Browse your files**.  After the package is uploaded, your package is validated.  After validation succeeds, review the extension details, and then select **Next** to continue.  If there are validation errors, resolve the issues and try uploading again.
+To upload the package, select and drag the package onto the upload field.  Also, you can select **Browse your files**.  After the package is uploaded, your package is validated.  After validation succeeds, review the extension details, and then select **Continue** to proceed.  If there are validation errors, resolve the issues and try uploading again.
```

Comment:
@ShashankMG123  I think some of this section needs to be numbered steps. I am also not sure about the button / link names. If you could provide raw screenshots at the "Save & Continue" steps at line 76, 100, and 117, I can verify the UI and possibly include one or more images as appropriate.

Reply:
Changes:
1. 
![image](https://user-images.githubusercontent.com/37444395/156703261-132ef371-9e25-47ba-a44d-4645c29d9ccf.png)
This has been changed from "Next" to "Continue"
2.
![image](https://user-images.githubusercontent.com/37444395/156703351-662387a0-edb7-45b2-91bb-7f40bf56e4b0.png)
Notice the buttons on the command bar. The "Save & Continue" button was previously a "Save draft" button

Brief about the new flow:
1. Previously there were situations where the Publish button was disabled even if all required data was filled (due to multiple reasons for example if there was a publish in progress) and we did receive feedback where the user did not understand why the button was disabled. In the new flow the publish button will be enabled as long as all required fields are completed. If there are any reason which prevents the user from publishing we will be surfacing the reason post the button click.
2. We have also made it easier for the user to publish an update by just clicking the publish button (previously the requirement was the click save draft and then publish.)

Let me know if this requires some sort of demo from my end to make the flow more clear.